src/services/OnboardingService.js

```diff
@@ -17,8 +17,9 @@ export default {
         context.$store.dispatch("user/firstDashboardVisit", true);
         context.$router.replace("/");
```

Comment:
While we're at it, could you move this redirect to be in the ActionView if the verification is successful? Not something that a service should be doing